Tasks/VsTest/taskinputparser.ts

```diff
@@ -33,6 +34,11 @@ export function getDistributedTestConfigurations() {
     }
     tl._writeLine(tl.loc('dtaNumberOfAgents', dtaConfiguration.numberOfAgentsInPhase));
 
+    const useVsTestConsole = tl.getVariable('UseVsTestConsole');
+    if (useVsTestConsole) {
+        dtaConfiguration.useVsTestConsole = useVsTestConsole;
+    }  
```

Comment:
code is fine. did we close on this to make the new flow off by default

---
In reply to: [128434633](https://github.com/Microsoft/vsts-tasks/pull/4832#discussion_r128434633) [](ancestors = 128434633)